src/main/java-templates/org/postgresql/Driver.java

```diff
@@ -411,7 +411,7 @@ public Connection getResult(long timeout) throws SQLException {
      * @throws SQLException if the connection could not be made
      */
     private static Connection makeConnection(String url, Properties props) throws SQLException {
-        return new @JDBCCONNECTCLASS@(hostSpecs(props),
+        return new ${JDBCCONNECTCLASS}(hostSpecs(props),
```

Comment:
@vlsi The stress one has build and debug the project inside of the IDE. At least Eclipse's Maven plugin (m2e) is not particularly well in that discipline. About makeConnection: You cannot derive from the system which connection type to create, as this is not necessarily bound to the platform. For example, the debugging platform might be JDK 8 (i. e. JDBC 4.2) but what you like to debug is JDBC 4. Also I don't like the idea that something that can be fixed at compile time is deferred to run time; this smells like error-prone and unnecessary runtime overhead. About not dropping something: Well, we can EVERYTHING keep as it is... but isn't improvement about CHANGE? ;-) And yes, at least I personally care about the cost of createConnection really. :-)